### PR TITLE
Clint tailwind

### PR DESCRIPTION
**Pull request by mrclintoka**
deleted in-line styles. and updated post-content headers

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -5,7 +5,17 @@ module.exports = {
...
+      fontFamily: {
+        'oswald': ['"Oswald"', "sans-serif"]
+      }
+    },
+    backgroundColor: {
+      primary: "#222b64",
+      gray: "#1f2937",
+      lightGray: "#d1d5db",
+      slate: "#cbd5e1"
+    }
```
Comment (RoskiDeluge) [inline tailwind.config.js:18]: Great job figuring out how to accomplish this!

File: components/posts/post-item.js [hunk 2]
```diff
@@ -37,13 +34,9 @@ function PostItem(props) {
...
           </div>
-          <div className="bg-gray-800 p-4 h-52 flex flex-col items-center"
+          <div className="bg-gray p-4 h-52 flex flex-col items-center"
           >
-            <h3 className="text-white text-2xl text-extrabold mb-5"
-            style={{
-              fontFamily: 'Oswald',
-              fontSize: '2rem'
-              }}
+            <h3 className="text-white text-4xl font-oswald text-extrabold mb-5"
```
Comment (RoskiDeluge) [inline components/posts/post-item.js:39]: This is really cool. It's just rad to see how immediate the code gets "leaner" as soon as you imported the font and used the 'font-oswald' property.

File: components/layout/main-navigation.js [hunk 3]
```diff
@@ -4,13 +4,7 @@ import Logo from './logo';
...
 function MainNavigation() {
   return (
-    <header className="flex justify-between bg-white p-6 place-items-center"
-    style={{ 
-        backgroundColor: "#222b64",
-        paddingLeft: "140px",
-        paddingRight: "140px",
-  
-      }}
+    <header className="flex justify-between bg-primary p-6 place-items-center px-36"
```
Comment (RoskiDeluge) [inline components/layout/main-navigation.js:7]: Nice job here. Lean code!! Awesome.

**Review by RoskiDeluge — APPROVED** [review]
Excellent job.

**Comment by RoskiDeluge** [comment]
Merging changes.

**Comment by mrclintoka** [comment]
Thanks Roberto :)

**Pull request merged**